src/commands/TruffleCommands.ts

```diff
@@ -241,6 +241,16 @@ export namespace TruffleCommands {
     }
     Telemetry.sendEvent("TruffleCommands.getPrivateKeyFromMnemonic.commandFinished");
   }
+  
+  export async function createContract(uri: Uri): Promise<void> {
```

Comment:
Hy @NikolaiSch,

Suggestion: I think we could try to put the contract inside the contract directory instead of root directory, and after that, do a refresh on Explorer box :)